tests/misc/test_dialogs.py

```diff
@@ -184,16 +184,10 @@ def test_updated_fact_same(self, dummy_window, fact):
         Make sure the property returns Fact with matching field values.
 
         We need to jump through some extra hoops because we the current
```

Comment:
What hoops? “the current implementation will always set the edited fact to today” Will it?